src/main/java/bizbook/logic/commands/AddNotesCommand.java

```diff
@@ -60,7 +59,7 @@ public CommandResult execute(Model model) throws CommandException {
         Person personToEdit = lastShownList.get(index.getZeroBased());
 
         // Update notes with new note
-        Set<Note> notesToEdit = new LinkedHashSet<>(personToEdit.getNotes());
+        ArrayList<Note> notesToEdit = new ArrayList<>(personToEdit.getNotes());
```

Comment:
Variable name should be notesList, as this is a list object while noteToAdd refers to a single note object.